app/src/main/java/com/truongdc21/quickquotes/Presenter/FragmentFavorite/FavoriteFragmentPresenter.kt

```diff
@@ -1,4 +1,107 @@
 package com.truongdc21.quickquotes.presenter.fragmentFavorite
 
-class FavoriteFragmentPresenter {
+import android.content.Context
+import com.truongdc21.quickquotes.R
+import com.truongdc21.quickquotes.data.model.Author
+import com.truongdc21.quickquotes.data.model.Quotes
+import com.truongdc21.quickquotes.data.model.Tag
+import com.truongdc21.quickquotes.data.repository.AuthorRepository
+import com.truongdc21.quickquotes.data.repository.QuotesRepository
+import com.truongdc21.quickquotes.data.repository.TagRepository
+import com.truongdc21.quickquotes.data.source.local.OnLocalResultListener
+import java.lang.Exception
+
+class FavoriteFragmentPresenter (
+    private val context: Context,
```

Comment:
```suggestion
    private val mContext: Context,
```
thêm prefix``m`` luôn cho nó đồng bộ e!